libcontainer/rootfs_linux.go

```diff
@@ -377,9 +390,12 @@ func doTmpfsCopyUp(m *configs.Mount, rootfs, mountLabel string) (Err error) {
 	})
 }
 
+// Now takes a mountConfig, we need to add the mountFile *os.File to that
+// struct, set it in the proper places and use that instead :(
```

Comment:
This looks like a TODO item -- why it's not addressed?

Reply:
Ouch, it was a TODO item when rebasing, but I missed the "TODO" and forgot to delete it :see_no_evil: 

Will delete it now, thanks for catching this :)